src/Compilers/CSharp/Test/Semantic/Semantics/TopLevelStatementsTests.cs

```diff
@@ -5268,14 +5266,14 @@ public void ExplicitMain_10()
             var text = @"
 System.Console.Write(42);
 
-partial class Program
+partial class Program2
```

Comment:
>Program2 [](http://example.com/codeflow?start=14&length=8)

It looks like this doesn't quite restore the original test scenario. It was significantly modified in the previous PR.